CHANGELOG.md

```diff
@@ -8,6 +8,54 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.0] - 2024-03-01
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.4.3...v1.5.0)
+
+### Added 
+
+- OpenTelemetry tracing and attributes to check algorithm ([#1331](https://github.com/openfga/openfga/pull/1331), [#1388](https://github.com/openfga/openfga/pull/1388))
+- Dispatch count to check response metadata as a query complexity heuristic ([#1343](https://github.com/openfga/openfga/pull/1343))
+
+### Fixed
+
+- Fix incorrect path for gPRC health check ([#1321](https://github.com/openfga/openfga/pull/1321))
+
+### Breaking Change :warning:
+
+The `AuthorizationModelReadBackend` interface method `FindLatestAuthorizationModelID` has changed to `FindLatestAuthorizationModel` for performance improvements. [#1387](https://github.com/openfga/openfga/pull/1387)
+
+If you implement your own data store, you will need to make the following change:
+
+<table>
+<tr>
+<th>Before</th>
+<th>After</th>
+</tr>
+<tr>
+<td>
+
+```go
+func (c *cachedOpenFGADatastore) FindLatestAuthorizationModelID(ctx context.Context, storeID string) (string, error) {
```

Comment:
Let's drop the `(c *cachedOpenFGADatastore)` bit. It's a little misleading.

Can we do `func ... FindLatestAuthorizationModelID(...)` instead?